Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "halo2"
-version = "0.1.0-beta.1"
+version = "0.2.0-beta.1"
```

Comment:
This should not have been altered. We have not cut an `0.1.0` release yet, and altering the MSRV during development is perfectly acceptable.